homeassistant/components/zwave.py

```diff
@@ -92,6 +93,10 @@
      [COMMAND_CLASS_DOOR_LOCK],
      TYPE_BOOL,
      GENRE_USER),
+    ('rollershutter',
+     [COMMAND_CLASS_SWITCH_MULTILEVEL],
```

Comment:
I was looking at the specific command class last night. Excellent work😊.It looks like a good idea for the future too. Feel free to make a pull request against my branch, as I won't have time till Saturday to change this. With the code you have at the Base of zwave.py it should be easier to filter components.👍